Integrations/integration-GoogleVault.yml

```diff
@@ -471,7 +554,7 @@ script:
                 'Contents': matters,
                 'HumanReadable': markdown,
                 'EntryContext': {
-                    'GoogleVault.Matter(val.ID == obj.ID)': context_output
+                    'GoogleVault.MatterList': context_output
```

Comment:
Why is there no unique field, like `MatterID` ?

Reply:
Question is if I get the object in the first time, and then I get different data in the next time, do we want it to be under the same object in the context?

Reply:
Yes defiantly. As this is a list of relevant matters the only situation where the list would change is if a matter was added / deleted. In both cases we would like to have the most updated list and overwrite the previous one.

Reply:
If you don't have an comparing id, then even if you get the same matter, it will create new matter in the context and now you will have 2 matter with the same id. 
What is the difference between Matter and MatterList? Just the name?

Reply:
And also I don't in any place in the outputs MatterList, just Matter

Reply:
Fixed